pkg/drivemgr/drivemgr-update-history.md

```diff
@@ -0,0 +1,7 @@
+### drivemgr update history
+
+drivemgr have several implementations, file here to track changeset to create new
+git commit for proprietary code.
+
+2023/04/07  proprietary code change, tracked by internal number 25693.
```

Comment:
This line is for rel/objectscale-1.2 branch commit. In our master branch commit, we don't have this.
On the other hand, for the next line, it would be better for us to make the date here corresponding to the commit in the private code?